pkg/kvs/kvs.go

```diff
@@ -34,10 +40,18 @@ func (s *keyValueStore) Get(key []byte) ([]byte, error) {
 }
 
 func (s *keyValueStore) Put(key []byte, value []byte) error {
-	return s.manifest.Add(context.Background(), hex.EncodeToString(key), manifest.NewEntry(swarm.NewAddress(value), map[string]string{}))
+	err := s.manifest.Add(context.Background(), hex.EncodeToString(key), manifest.NewEntry(swarm.NewAddress(value), map[string]string{}))
+	if err != nil {
+		return err
+	}
+	s.putCnt++
+	return nil
 }
 
 func (s *keyValueStore) Save() (swarm.Address, error) {
+	if s.putCnt == 0 {
```

Comment:
Should Save() reset putCnt to 0 ? Otherwise it can be called without new content, if saved once.

Reply:
fixed: https://github.com/Solar-Punk-Ltd/bee/commit/84569406c1d0163cc1daba6524e356d79f2a3fb6